lib/rucio/daemons/bb8/bb8.py

```diff
@@ -96,9 +96,11 @@ def run_once(
         "bb8", "max_rse_rebalance_volume", default=500 * 1e9
     )
     min_total = config_get_float("bb8", "min_total", default=20 * 1e9)
+    total = config_get("bb8", "total_source", default="static")
```

Comment:
I think `total_source` is better than `total` as the variable name as you had it earlier. And then change the default to `"total"` so as not to change the behavior for others. I will put "static" into our configuration.